looper/conductor.py

```diff
@@ -8,8 +8,8 @@
 from copy import copy, deepcopy
 from json import loads
 from subprocess import CalledProcessError, check_output
+from typing import *
```

Comment:
I noticed that you are importing everywhere everything from typing. Just wondering if it is better way to do so? Because every code that I was working on before it was written that way: from typing import (List, ...).

Reply:
This can be seen as a specific instance, I think, of the debate about implicit vs. explicit, with a specific set of tradeoffs. By `import *`, we risk loss of clarity about where certain names come from, and we're more vulnerable to name collisions from various imports; on the other hand, we gain things like making it easier to use name names from whatever package we're importing from (without need to explicit import new things), and safety that we haven't used a name we've forgotten to import (which would be especially bad, for example, if we introduce new code that's not covered by tests, and then release software that can hit a `NameError` when a user path through the code touches what wasn't executed by the tests and where we've forgotten to import a name we've used.

In general, I favor explicit over implicit, especially in Python where it's more the "norm," but here is an exception I think, namely because `typing` is part of the standard library, and the fact that the names we're likely to use from it are the capitalized ones and ones with which many readers will be familiar. As such, I think we lose little by `import *`, but we still gain its conveniences. We can discuss more in the meeting if you want :)